src/sugar.jl

```diff
@@ -1198,11 +1195,14 @@ function autodiff(
     end
 end
 
+batchify_activity(::Type{Active{T}}, ::Val{B}) where {T,B} = Active{T}
+batchify_activity(::Type{Duplicated{T}}, ::Val{B}) where {T,B} = BatchDuplicated{T,B}
```

Comment:
Internal use only? Or does it make sense to move to EnzymeCore?

Reply:
I can move it to EnzymeCore

Reply:
Do you want it documented and public? If so, did I forget any cases? Like what do we do with `DuplicatedNoNeed` and other variants?

Reply:
Sofar is it documented but not exported